single-pool/js/tests/fixtures/spl_single_validator_pool.so

```diff
@@ -0,0 +1 @@
+../../../../target/deploy/spl_single_validator_pool.so
```

Comment:
Is it possible to specify another search directory in bankrun?

Reply:
i can set an environment var but doing it in js seemed hacky and doing it in the github script would break testing locally

Reply:
Ah gotcha, a link makes sense then